src/main/java/com/avispl/symphony/dal/infrastructure/management/qsc/qsysreflect/QSysReflectCommunicator.java

```diff
@@ -334,7 +429,7 @@ protected void internalInit() throws Exception {
 		apiToken = this.getPassword();
 		this.setBaseUri(QSysReflectConstant.QSYS_BASE_URL);
 		if (checkValidApiToken()) {
-			executorService = Executors.newSingleThreadExecutor();
+			executorService = Executors.newFixedThreadPool(8);
```

Comment:
is this the most optimal number for the aggregator? Are there any API limitations on number of requests per second?